src/app/core/admin/services/child-photo-update.service.ts

```diff
@@ -42,7 +42,8 @@ export class ChildPhotoUpdateService {
       Child.generatePhotoPath(filename)
     );
     if (fileExists) {
-      child.photoFile = filename;
+      const currentPhoto = child.photo;
+      child.photo = { path: filename, photo: currentPhoto?.photo };
```

Comment:
why should `photo.photo` still be the "old" photo rather than the new filename here?

Reply:
To be honest, I dont really know what this function does and when it is supposed to be used but what I have done is equivalent to the current behavior where only the filename is updated but the `BehaviorSubject` is not changed.